api-reference/10 UI Components/dxSelectBox/1 Configuration/fieldTemplate.md

```diff
@@ -17,10 +17,106 @@ The selected item's data.
 A template name or container.
 
 ---
+If you define a **fieldTemplate**, the component does not render the underlying HTML. In this case, manually render hidden input with the corresponding name attribute to submit values through a HTML form.
+
+The example below shows how to define the **fieldTemplate**. If you want to customize field text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```

Comment:
`to customize field text` A field? or fields in general?

Reply:
A component (SelectBox or TagBox) has a field, I'm talking about such components in general.

Reply:
```suggestion
The example below shows how to define the **fieldTemplate**. If you want to customize this field's text, use the TextBox's [text](/Documentation/ApiReference/UI_Components/dxTextBox/Configuration/#text) property.
```